### DISCUSSION TITLE
Intune Policy

### DISCUSSION THREAD

**Discussion by narendrapuranik** [community/community#129985] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How to create policy for admin password protection to install apps on windows system

**Comment by ebndev** [comment]
Hi @narendrapuranik, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about what you're trying to accomplish. Giving a few more details might help someone give you a nudge in the right direction. 😄
